### ISSUE TITLE
Guidelines to group members failing incoming message processing

### ISSUE THREAD

**Issue by marco-tiloca-sics** [ace-wg/ace-key-groupcomm#147]
Section 4.8.1.1 "Retrieve Group and Individual Keying Material"

TL;DR: Provide more guidelines for a group member that keeps failing to decrypt/verify incoming messages, even after having retrieved the latest keying material from the KDC. The group member can try to retrieve the authentication credentials of the message senders. If that still does not work, the group member can try to retrieve the authentication credential of the KDC.

Following the multiple failed decryption and verification of an incoming message and the retrieval of the latest group keying material, the group member may retrieve the group keying material that it is already storing.

In such a case, the failed decryption might be due to the message sender and/or the KDC that has changed its authentication credential.

Hence, the group member can follow-up by retrieving the authentication credential of the message sender (see Section 4.4). If such an authentication credential has not changed compared to the currently stored one, then the group member can further follow-up by retrieving the authentication credential of the KDC (see Section 4.5).
